roles/tpa_single_node/templates/manifests/bombastic/indexer/Deployment.yaml.j2

```diff
@@ -1,3 +1,4 @@
+apiVersion: apps/v1
```

Comment:
Is it needed ?

Reply:
Not 100% sure it makes a difference, I thought it does, but we need to be consistent, so let's keep it for now and double check to eventually remove it later.